src/main/java/main_objects/CollectionManager.java

```diff
@@ -0,0 +1,133 @@
+package main_objects;
+
+import exceptions.IDTakenException;
+import exceptions.LogException;
+import exceptions.WrongDataException;
+import exceptions.WrongKeyException;
+import io_utilities.LogUtil2;
+import io_utilities.Printer;
+import io_utilities.working_with_csv_file.CSVReader;
+import io_utilities.working_with_csv_file.CSVWriter;
+import io_utilities.working_with_input.InputChecker;
+import io_utilities.working_with_input.InputReader;
+import packets.Request;
+
+import java.io.IOException;
+import java.util.LinkedList;
+
+public class CollectionManager {
+    private final LinkedList<Flat> collection = new LinkedList<>();
+    private final String fileName;
+    private int id;
+
+    public CollectionManager(String fileName) {
+        this.fileName = fileName;
+    }
+
+    public void info() {
+        Printer.printResult("Type of DS: LinkedList");
+        Printer.printResult("Number of elements: " + collection.size());
+    }
```

Comment:
Why does your collection manager handle IO? Maybe move this away to the command?

Collection manager should not know anything about input and output